include/seqan/gff_io/gff_io_base.h

```diff
@@ -349,9 +349,15 @@ void readRecord(GffRecord & record, CharString & buffer, TFwdIterator & iter)
 {
     IsNewline isNewline;
 
-    clear(record);
+    if (value(iter) == '#') // skip this line and find another record
+    {
+        skipLine(iter);
+        readRecord(record, buffer, iter);
```

Comment:
@weese Thank you for the comment! Please feel free to let me know for other considerations.